### PR TITLE
[bump] client: 2.0.0-internal.4.0.0 => 2.0.0-internal.5.0.0 (major)

### PR DESCRIPTION
**Pull request by tylerbutler**
Bumped client from 2.0.0-internal.4.0.0 to 2.0.0-internal.5.0.0 as part of releasing 2.0.0-internal.4.0.0.

Commands used:

```shell
flub bump client -t major
```

### CODE REVIEW

File: examples/apps/collaborative-textarea/package.json [hunk 1]
```diff
@@ -38,16 +38,16 @@
 		"webpack:dev": "webpack --env development"
 	},
 	"dependencies": {
-		"@fluid-example/example-utils": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
-		"@fluid-experimental/react-inputs": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
-		"@fluidframework/aqueduct": ">=2.0.0-internal.4.0.0 <2.0.0-internal.5.0.0",
+		"@fluid-example/example-utils": "^2.0.0-internal.5.0.0",
```
Comment (CraigMacomber) [inline examples/apps/collaborative-textarea/package.json:41]: I think we want to keep "<" version ranges.

Comment (tylerbutler) [reply examples/apps/collaborative-textarea/package.json:41]: Good catch! I'm investigating to see if we have a bug in the bump tools.

Comment (tylerbutler) [reply examples/apps/collaborative-textarea/package.json:41]: Found the bug: #14727

**Review by scottn12 — APPROVED** [review]

**Pull request merged**